client/src/local_character_controller.rs

```diff
@@ -3,7 +3,7 @@ use common::{math, proto::Position};
 pub struct LocalCharacterController {
     /// The last extrapolated inter-frame view position, used for rendering and gravity-specific
     /// orientation computations
-    position: Position,
+    pub position: Position,
```

Comment:
As an alternative to making this public, you can also use the `oriented_position()` method. That method might have fewer surprises, such as making sure the orientation of the blinker you place matches the direction the player is facing.

Reply:
Nice. Will do.